wiki/Beatmap_Editor/Distance_Snap/en.md

```diff
@@ -1,3 +1,12 @@
+---
+tags:
+  - ds
+---
+
+```yaml
+needs_cleanup: true
+```
```

Comment:
shouldnt this be just
```suggestion
---
tags:
  - ds
needs_cleanup: true
---
```

Reply:
i was a bit lost on this, and was waiting for someone to tell me if i did it wrong. I'll do what ya suggested